default.ascx

```diff
@@ -1,67 +1,110 @@
-<!--#include file="partials/_registers.ascx" -->
-<!--#include file="partials/_includes.ascx" -->
+<%@ Control Language="c#" AutoEventWireup="false"
+    Explicit="True" Inherits="DotNetNuke.UI.Skins.Skin" %>
+<%@ Register TagPrefix="dnn" TagName="LOGIN" Src="~/Admin/Skins/Login.ascx" %>
+<%@ Register TagPrefix="dnn" TagName="LOGO" Src="~/Admin/Skins/Logo.ascx" %>
+<%@ Register TagPrefix="dnn" TagName="TERMS" Src="~/Admin/Skins/Terms.ascx" %>
+<%@ Register TagPrefix="dnn" TagName="COPYRIGHT" Src="~/Admin/Skins/CopyRight.ascx" %>
+<%@ Register TagPrefix="dnn" TagName="PRIVACY" Src="~/Admin/Skins/Privacy.ascx" %>
+<%@ Register TagPrefix="dnn" TagName="BREADCRUMB" Src="~/Admin/Skins/BreadCrumb.ascx" %>
+<%@ Register TagPrefix="dnn" TagName="MENU" src="~/DesktopModules/DDRMenu/Menu.ascx" %>
+<%@ Register TagPrefix="dnn" TagName="LANGUAGE" Src="~/Admin/Skins/Language.ascx" %>
+<%@ Register TagPrefix="dnn" TagName="SEARCH" Src="~/Admin/Skins/Search.ascx" %>
+<%@ Register TagPrefix="dnn" TagName="USER" Src="~/Admin/Skins/User.ascx" %>
+<%@ Register TagPrefix="dnn" TagName="CURRENTDATE" Src="~/Admin/Skins/CurrentDate.ascx" %>
 
-<!-- Header/NavBar -->
-<!--#include file="partials/_header.ascx" -->
 
-<!-- Main Content -->
-<main>
-  <div class="container-fluid bannerpane px-0 mb-0">
-    <div class="justify-content-center">
-      <div id="BannerFluidPane" class="text-center" runat="server"></div>
-    </div>
-  </div>
+<%@ Register TagPrefix="dnn" Namespace="DotNetNuke.Web.Client.ClientResourceManagement" Assembly="DotNetNuke.Web.Client" %>
+<%@ Register TagPrefix="fortyfingers" TagName="STYLEHELPER" Src="~/DesktopModules/40Fingers/SkinObjects/StyleHelper/StyleHelper.ascx" %>
 
-  <div class="container py-4">
-    <div class="row justify-content-center">
-      <div id="ContentPane" class="col-md-12" runat="server"></div> 
-    </div>
-  </div>
 
-  <div class="container">
-    <div class="row">
-      <div id="LeftHalfPane1" class="col-md-6" runat="server"></div>
-      <div id="RighttHalfPane1" class="col-md-6" runat="server"></div>
-    </div>
-  </div> 
-  
-  <div class="container">
-    <div class="row">
-      <div id="WideLeftPane" class="col-lg-8 cpr-5" runat="server"></div>
-      <div id="NarrowRightPane" class="col-lg-4 bg-light-grey cp-5" runat="server"></div>
-    </div>
-  </div>   
+	<!-- FontAwsome 5 + 4 shim-->
+	<dnn:DnnCssInclude runat="server" FilePath="https://cdn.jsdelivr.net/npm/@fortawesome/fontawesome-free@5.15.4/css/all.min.css" Name="fontawesome" Version="5.15.4" Priority="15"  />
```

Comment:
This should be managed by npm for better maintainability.  Also, it looks like it is also being loaded in at least one other place in the theme.